File: abc/abc122/abc122_a.cs
// https://atcoder.jp/contests/abc122/submissions/9265582

using System;
using System.Collections.Generic;

namespace CSharp
{
    class Program
    {
        static void Main()
        {
            var c = new Dictionary<char, char>()
            {
                {'A', 'T'},
                {'T', 'A'},
                {'G', 'C'},
                {'C', 'G'}
            };

            var b = Console.ReadLine().ToCharArray();

            Console.WriteLine(c[b[0]]);
        }
    }
}

File: abc/abc122/abc122_a.fs
// https://atcoder.jp/contests/abc122/submissions/12205438

stdin.ReadLine()
|> fun b ->
    match b with
    | "A" -> "T"
    | "C" -> "G"
    | "G" -> "C"
    | "T" -> "A"
    | _ -> ""
|> stdout.WriteLine
